Scripts/script-MatchRegex.yml

```diff
@@ -6,24 +6,32 @@ script: |-
   var data = (typeof args.data) === 'string' ? args.data : JSON.stringify(args.data);
   var flags = args.flags ? args.flags : 'im';
   var r = new RegExp(args.regex, flags);
-  var m = r.exec(data);
-  if (m && m.length > 0) {
-      var val = m[0];
-      var group = parseInt(args.group);
-      if (args.group && m.length > group) {
-          val = m[group];
+  var m;
+  var vals = [];
+  while ((m = r.exec(data)) !== null) {
```

Comment:
!MatchRegex_copy data="asd" regex="asd" causes infinite loop, i suspect it happens because r.exec() always returns a value without the 'g' flag.

Reply:
I've fixed this with the latest changes